### DISCUSSION TITLE
2021-04-15 阅读《图形思考》的意外收获 ~ 蒋继发 (ThaddeusJiang)

### DISCUSSION THREAD

**Discussion by giscus** [ThaddeusJiang/thaddeusjiang.github.io#32] [Announcements]
# 2021-04-15 阅读《图形思考》的意外收获 ~ 蒋继发 (ThaddeusJiang)

去年看过一本《图解即战力》的书，非常赞同，但是并没有在实际生活和工作中实践。主要原因就是不知道如何下手。如果说《图解即战力》是一本侧重理论，那么《图形思考》就更侧重实践。

https://thaddeusjiang.com/2021-04-15-yue-du-tu-xing-si-kao-de-yi-wai-shou-huo.html

**Comment by htoooth** [comment]
图解即战力 没有找到这本书，能给个链接吗？

**Reply by ThaddeusJiang** [reply]
@htoooth 等我找找，不好意思，可能是名字记错了。

**Reply by ThaddeusJiang** [reply]
@htoooth sorry, 这么晚才回复。

我应该是记错名字了，日本的书名太长了。十分抱歉。

[《エンジニアのための新教養 □○△で描いて、その場でわかるシンプル図解 何でも伝え、何でもまとめるストラクチャードコミュニケーション》](https://www.amazon.co.jp/%E3%82%A8%E3%83%B3%E3%82%B8%E3%83%8B%E3%82%A2%E3%81%AE%E3%81%9F%E3%82%81%E3%81%AE%E6%96%B0%E6%95%99%E9%A4%8A-%E2%96%A1%E2%97%8B%E2%96%B3%E3%81%A7%E6%8F%8F%E3%81%84%E3%81%A6%E3%80%81%E3%81%9D%E3%81%AE%E5%A0%B4%E3%81%A7%E3%82%8F%E3%81%8B%E3%82%8B%E3%82%B7%E3%83%B3%E3%83%97%E3%83%AB%E5%9B%B3%E8%A7%A3-%E4%BD%95%E3%81%A7%E3%82%82%E4%BC%9D%E3%81%88%E3%80%81%E4%BD%95%E3%81%A7%E3%82%82%E3%81%BE%E3%81%A8%E3%82%81%E3%82%8B%E3%82%B9%E3%83%88%E3%83%A9%E3%82%AF%E3%83%81%E3%83%A3%E3%83%BC%E3%83%89%E3%82%B3%E3%83%9F%E3%83%A5%E3%83%8B%E3%82%B1%E3%83%BC%E3%82%B7%E3%83%A7%E3%83%B3-%E5%8A%A0%E5%B3%B6-%E4%B8%80%E7%94%B7/dp/4798171271/ref=sr_1_7?__mk_ja_JP=%E3%82%AB%E3%82%BF%E3%82%AB%E3%83%8A&crid=3KC13XJUMV86R&keywords=%E5%9B%B3%E8%A7%A3&qid=1678692016&sprefix=%E5%9B%B3%E8%A7%A3+%E5%8D%B3%E6%88%A6%E5%8A%9B,aps,184&sr=8-7)
